<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketForm</class>
 <widget class="QWidget" name="SocketForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QTextEdit" name="ReadMsg"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>服务器IP:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="ServerIP"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>端口号：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="port"/>
   </item>
   <item row="2" column="0" rowspan="3">
    <widget class="QTextEdit" name="SendMsg"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="btnconnect">
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="btnDisconnect">
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="btnSend">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
